From 7079b1bae1eaf3e43ed13c5e8b28c18df5335e40 Mon Sep 17 00:00:00 2001
From: Jeffy Chen <jeffy.chen@rock-chips.com>
Date: Tue, 19 Jan 2021 10:25:38 +0800
Subject: [PATCH 01/14] Revert "GStreamer: Update render rect only if open gl
 sink is used"

This reverts commit a553c7d1b63defc963a043e363f8d75d255e9c72.

We are using render rect in waylandsink and kmssink too.

Signed-off-by: Jeffy Chen <jeffy.chen@rock-chips.com>
---
 src/gsttools/qgstreamervideowidget.cpp | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/src/gsttools/qgstreamervideowidget.cpp b/src/gsttools/qgstreamervideowidget.cpp
index 4137aff..164e62f 100644
--- a/src/gsttools/qgstreamervideowidget.cpp
+++ b/src/gsttools/qgstreamervideowidget.cpp
@@ -38,7 +38,6 @@
 ****************************************************************************/
 
 #include "qgstreamervideowidget_p.h"
-#include "qgstutils_p.h"
 
 #include <QtCore/qcoreevent.h>
 #include <QtCore/qdebug.h>
@@ -171,8 +170,7 @@ bool QGstreamerVideoWidgetControl::eventFilter(QObject *object, QEvent *e)
 
         if (e->type() == QEvent::Paint) {
             // Update overlay by new size if any.
-            if (QGstUtils::useOpenGL())
-                m_videoOverlay.setRenderRectangle(QRect(0, 0, m_widget->width(), m_widget->height()));
+            m_videoOverlay.setRenderRectangle(QRect(0, 0, m_widget->width(), m_widget->height()));
             if (m_videoOverlay.isActive())
                 m_videoOverlay.expose(); // triggers a repaint of the last frame
             else
-- 
2.20.1

